src/test/user.spec.js

```diff
@@ -674,3 +675,115 @@ describe('PATCH /api/v1/auth/user/:id/role', () => {
     expect(userRoleStatus.length).to.equal(1);
   });
 });
+
+describe('/api/v1/auth/reset-password', async () => {
+  await db.User.create({
```

Comment:
We already have a method that creates users in the `test/utils/helpers.js/`. You want to check it out and use it on your test. [test/utils/helpers.js](https://github.com/andela/ah-kifaru-backend/blob/develop/src/test/utils/helpers.js)

Reply:
Also, try to make use of the methods in our `base.repository` and not the direct sequelize method.